src/components/Column/Column.tsx

```diff
@@ -44,7 +44,7 @@ export default function Column({
           <ColumnBody tickets={column.tickets} onEditClick={onEditClick} />
         )}
 
-        <ColumnFooter onAddTicket={onAddTicket} />
+        <ColumnFooter onAddTicket={handleAddTicket} />
```

Comment:
It feels like `onAddTicket` can be embedded in `ColumnFooter` itself so that you can reduce lines in this parent component. If you like putting API calls in this file, it's also okay for me too.

Reply:
yes that is my intention.